pkg/mkimage-raw-efi/make-raw

```diff
@@ -216,8 +222,8 @@ do_imga() {
 }
 
 do_imgb() {
-    # for now we are not initializing IMGB - hence passing /dev/null
-    do_rootfs $1 IMGB /dev/null
+    # for now we are not initializing IMGB - hence not passing the source
```

Comment:
Instead of "not initializing" how about "wiping"?

Reply:
make sense, modified